resnet-burn/Cargo.toml

```diff
@@ -11,16 +11,13 @@ default = ["burn/default"]
 
 [dependencies]
 # Note: default-features = false is needed to disable std
-burn = { version = "0.12.1", default-features = false}
-# burn = { path = "../../burn/burn", default-features = false}
+burn = { path = "../../burn/burn", default-features = false}
 serde = { version = "1.0.192", default-features = false, features = [
     "derive",
     "alloc",
 ] } # alloc is for no_std, derive is needed
 
 [dev-dependencies]
-burn = { version = "0.12.1", default-features = false, features = ["ndarray"]}
-burn-import = "0.12.1"
-# burn = { path = "../../burn/burn", default-features = false}
-# burn-import = {path = "../../burn/burn-import"}
-# image = { version = "0.24.7", features = ["png", "jpeg"] }
\ No newline at end of file
+burn = { path = "../../burn/burn", default-features = false} # local burn to include the with_key_remap chain fix
```

Comment:
I don't think we need to specify burn in both dev and normal dependencies.

Reply:
Whoops, you're right.